cog/slots.py

```diff
@@ -55,16 +58,39 @@ class Slots(customs.cog.Cog):
         }
 
 
+    _credits = {
+        'low' : 1,
+        'mid' : 5,
+        'high' : 15
+    }
+
+
+    # credits -> low = 1, mid = 5, high = 15
     @commands.group(alias=['slots'])
-    async def slots(self, ctx, credits):
+    async def slots(self, ctx, credits_index = 'low'):
         '''
         Runs the slot machine.
         '''
+        if credits_index not in Slots._credits:
+            await ctx.send("BAKA")
+            return
+
+        consumer = ctx.message.author
+        consumer_data = db_user_interface.fetch(self.bot.db_user, consumer.id)
+        consumer_frogs = consumer_data['frogs_normal']
+
+        if Slots._credits[credits_index] > consumer_frogs:
+            embed = make_simple_embed('', 'You do not have enough frogs! You only have **`{count}`**'.format(count=consumer_frogs))
+            await ctx.send(content=consumer.mention, embed=embed)
+            return
+
+        db_user_interface.modify_frog(self.bot.db_user, ctx.message.author.id, -Slots._credits[credits_index])
```

Comment:
Kind of dangerous at the moment.  db_user_interface.modify_frog() doesn't check to make sure that the result is negative frogs. You did a checked it earlier, but just take notes.